charts/gonic/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Default values for Gonic.
+
+image:
+  repository: sentriz/gonic
+  pullPolicy: IfNotPresent
```

Comment:
Not sure if defaulting to `Always` here might be better for charts that use a `latest` tag by default?

Reply:
The devs will be adding semver in their next release, so I can update the image tag in the future. 

https://github.com/sentriz/gonic/pull/112